sidecar/replication/go/index.tmpl

```diff
@@ -15,9 +15,9 @@
   <label for="master">Master</label><br><br>
   <input type="text" placeholder="Flag Name" name="flagName" id="flagName" /><br><br>
```

Comment:
```suggestion
  <input type="text" placeholder="Flag Key" name="flagKey" id="flagKey" /><br><br>
```

Cant we hardcode this value too?

Reply:
or make it a dropdown? i think we really only need 1 flag, not have them try 50 different ones